iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/aup/aup.create.dialog.html

```diff
@@ -22,19 +22,23 @@ <h3 class="modal-title">
 </div>
 <div class="modal-body">
     <form name="createAup">
-        <div class="form-group" ng-class="{'has-error': $ctrl.error || createAup.url.$dirty && !createAup.url.$valid}">
+        <div class="form-group has-error" ng-if="$ctrl.error">
+            <span class="help-block">
+                {{ $ctrl.error }}
+            </span>
+        </div>
+        <div class="form-group">
             <label>Acceptable Usage Policy URL</label>
-            <input name="url" class="form-control" type="url" value="http://" ng-model="$ctrl.aupVal.url" required="true">
```

Comment:
Why type="url" was a problem?

Reply:
Because `type="url"` expects a URL and and if we add a string in the form it is ignored and thus we get a wrong error from the url validator, i.e. I`nvalid AUP: the AUP URL cannot be blank` instead of `Invalid AUP: the AUP URL is not valid`